src/mjolnir/util.cc

```diff
@@ -130,14 +129,33 @@ bool shapes_match(const std::vector<PointLL>& shape1, const std::vector<PointLL>
   }
 }
 
+bool load_spatialite(sqlite3* db_handle) {
```

Comment:
doing this makes valhalla build out of the box on all the latest versions of ubuntu up to 20.04